src/dotnet/CarbonAware.WebApi/Program.cs

```diff
@@ -13,11 +13,19 @@
 builder.Services.Configure<CarbonAwareVariablesConfiguration>(builder.Configuration.GetSection(CarbonAwareVariablesConfiguration.Key));
 builder.Services.AddCarbonAwareEmissionServices(builder.Configuration);
 builder.Services.AddCarbonAwareSciScoreServices(builder.Configuration);
+CarbonAwareVariablesConfiguration config = new CarbonAwareVariablesConfiguration();
+
+builder.Configuration.GetSection(CarbonAwareVariablesConfiguration.Key).Bind(config);
 
 builder.Services.AddHealthChecks();
 
 var app = builder.Build();
 
+if (!string.IsNullOrWhiteSpace(config.WebApiRoutePrefix))
+{
+    app.UsePathBase(config.WebApiRoutePrefix);
```

Comment:
I was wondering whether you can specify something like "../../../../../". Not sure if that is what @gfmatthews tried to bring it up here.